src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -22,28 +23,36 @@
    */
   private final CANSparkMax m_drumMotor;
   private final CANSparkMax m_feedWheelMotor;
-  private int m_repsAboveDrum = 0;
-  private int m_repsAboveFeed = 0;
+  private final Encoder m_drumEncoder;
+  private final Encoder m_feedWheelEncoder;
```

Comment:
The NEO motors have built in hall sensors which function similar to encoder, as such dedicated encoders are not necessary here. You can access the hall sensors with the `CANSparkMax.getEncoder()` method